framework/field_functions/field_function_grid_based.h

```diff
@@ -18,109 +18,78 @@ namespace opensn
 class SpatialDiscretization;
 class GhostedParallelSTLVector;
 
-/***/
 class FieldFunctionGridBased : public FieldFunction
 {
 public:
   typedef std::pair<Vector3, Vector3> BoundingBox;
 
-  /**Returns required input parameters.*/
   static InputParameters GetInputParameters();
-
-  /**ObjectMaker based constructor.*/
   explicit FieldFunctionGridBased(const InputParameters& params);
 
-  /**Creates a field function, filling it with zeros.*/
-  FieldFunctionGridBased(const std::string& text_name,
+  /// Creates a field function, filling it with zeros.
```

Comment:
Coding standard says to use `/** */` over `///`. But, I do see significant use of `///` mainly for single line comments. Maybe an update to the coding standard? What do you guys think?

My personal preference is:
```
/**
 * text
 */
method(params);

/// Description
type variable;
```
Because typically API needs more text that the variable. It is rarely the case where variables need more than 1 line and if so, multiline `///` does the trick.

Reply:
> Does `/** */` on one line fit within the coding standards?

It does. And it is used in the code quite a bit as well. If others agree, we can amend the coding standard to allow `///` for single line comments...

Reply:
I am fine with both `/** */~ and `///`